CONTRIBUTING.md

```diff
@@ -1,72 +1,49 @@
 
-Note as well that, as a community-driven open source project, we depend on our
-community to be able to maintain and develop AtoM. We are committed to
-including as many bug fixes as we can in each new release, but a confirmed
-report is not enough alone to guarantee that a fix will be included in the
-next release. If your institution is interested in sponsoring a fix, feel free
-to contact Artefactual Systems for an estimate - all sponsored development
-will be included in the next public release. For more information on how we
-develop and maintain AtoM, please see:
+Start by doing a search on [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
+and in the [AtoM GitHub issues](https://github.com/artefactual/atom/issues)
+to see if your question has been considered in the past.
 
-* https://wiki.accesstomemory.org/Development/Philosophy
+If you are sure that the issue you have is a bug, please open an issue 
+on GitHub. If not, feel welcome to open a thread in the Users Group forum.
 
-We also welcome pull requests to fix issues!
+The [Troubleshooting guide](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/)
+contains useful tips on how to find information needed to open an issue,
+such as the [version of AtoM](https://www.accesstomemory.org/docs/latest/user-manual/administer/settings/#application-version),
+accessing [errors logs and debug mode](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/#troubleshooting-logs-debug),
+and other information needed to [get support](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/#getting-support).
 
 ## Contributing code
 
-If you're considering contributing code to the project, please read our
-contribution page and familiarize yourself with our coding style and code
-review guidelines.
+Please read our contribution page and familiarize yourself with our coding
+style and code review guidelines.
 
-* Find out more about our Code Review process here: [Code review](https://wiki.accesstomemory.org/Development/Code_review)
-* Learn about our Coding standards here: [Coding standard](https://wiki.accesstomemory.org/Development/Coding_standard)
-* Find information about our Code repository here: [Code repository](https://wiki.accesstomemory.org/Resources/Code_repository)
-* Read about our Testing strategies here: [Testing](https://wiki.accesstomemory.org/Development/Testing)
+* Find out more about our [Code review](https://wiki.accesstomemory.org/Development/Code_review)
+* Learn about our [Coding standard](https://wiki.accesstomemory.org/Development/Coding_standard)
+* Find information about our [Code repository](https://wiki.accesstomemory.org/Resources/Code_repository)
+* Read about our [Testing strategies](https://wiki.accesstomemory.org/Development/Testing)
 
 We ask that all code contributors complete and return a signed Contributor’s
```

Comment:
is it possible to link a copy of the Contributor's Agreement here?

Reply:
I added a link to https://wiki.accesstomemory.org/wiki/Development/Contribute_code#Copyright_and_license